app/src/main/java/ip/team13/petowner/data/dto/pet/PetDataModel.kt

```diff
@@ -0,0 +1,43 @@
+package ip.team13.petowner.data.dto.pet
+
+import androidx.databinding.BaseObservable
+import androidx.databinding.Bindable
+import kotlin.random.Random
+import androidx.databinding.library.baseAdapters.BR
+
+data class PetDataModel(
+    val id: String,
+    val name: String?,
+    val groupId: String?,
+    val pictureUrl: String?,
+    @Transient
+    private var _isSelected: Boolean = false
```

Comment:
Can we ditch the `_` prefix as it doesn't follow the Kotlin recommended naming convention?